designer/server/src/common/components/sorting/template.njk

```diff
@@ -0,0 +1,49 @@
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% if params.sorting %}
+<form method="get" action="/library" aria-label="Sort forms">
```

Comment:
Mind if we remove this?

It has the same text as the button and the `<select>` + `<button>` make sense

Maybe a nice actual heading would be better still but it's not in the designs

```suggestion
<form method="get" action="/library">
```